.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
```

Comment:
Please, delete all unnecessary files from PR (u need to have only file with tests)

Reply:
Fixed